alf/nest/utils.py

```diff
@@ -230,8 +230,11 @@ def _traverse_transform(nested, levels):
             return nested._replace(**{level: new_val})
         elif isinstance(nested, dict):
             new_val = nested.copy()
-            new_val[level] = _traverse_transform(
-                nested=nested[level], levels=levels[1:])
+            _val = _traverse_transform(nested=nested[level], levels=levels[1:])
+            if _val is not None:
```

Comment:
This is unexpected behavior. What if the caller really want to transform it to None?
And with this behavior, None has different effect for namedtuple and dict. It seems not right.

Reply:
good point.  reverted.